command/agent/config.go

```diff
@@ -368,6 +423,11 @@ func DefaultConfig() *Config {
 		Addresses:      &Addresses{},
 		AdvertiseAddrs: &AdvertiseAddrs{},
 		Atlas:          &AtlasConfig{},
+		ConsulConfig: &ConsulConfig{
+			ServerServiceName: "nomad-server",
+			ClientServiceName: "nomad-client",
+			AutoRegister:      true,
```

Comment:
@dadgar That's the default the consul api client uses. Do you see any value in putting the default here too?